### PR TITLE
Update klog to 0.3.0

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:
This PR updates the `klog` dependency to the current revision which solves and issue where verbose flag wouldn't work as expected.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -30,6 +30,10 @@ required = [
   name="sigs.k8s.io/controller-tools"
   version="v0.1.9"
 
+[[constraint]]
+  name="k8s.io/klog"
+  revision="aaeac2fedbead93263ae78b80849cac7ff5a5fbb"
```
Comment (detiber) [inline Gopkg.toml]: Instead of locking the version here, should we push klog to release a new version and then just update the dep?

Comment (vincepri) [reply Gopkg.toml]: Already did, it seems they are blocked on a PR and they'll release after it's merged. I'll update once that happens :)

**Comment by vincepri** [comment]
Actually seems they might release sooner than expected, so let's hold this and I'll come back once klog version is released.

/hold

**Pull request merged**